src/input.js

```diff
@@ -38,6 +38,8 @@ const getBaseLcovFile = () => valueOrFalse(core.getInput("base-lcov-file"));
 const getSendSummaryComment = () =>
   valueOrFalse(core.getInput("send-summary-comment"));
 
+const getWorkspacePath = () => core.getInput("workspace-path");
```

Comment:
Can you add please this parameter to Readme file?

Reply:
@vitaliimelnychuk updated the readme